provider/provider.go

```diff
@@ -81,30 +79,21 @@ func (p *Provider) Schema(ctx context.Context, req tfprovider.SchemaRequest, res
 	resp.Schema = tfpschema.Schema{
 		Attributes: map[string]tfpschema.Attribute{
 			"url": tfpschema.StringAttribute{
-				Description: "URL of the API server.",
+				Description: fmt.Sprintf("URL sets the base URL of the API gateway. The default is `%s`", sdk.DefaultURL),
 				Optional:    true,
```

Comment:
maybe you can set here a default value?

Reply:
The default value is determined in the SDK level, no need to add it here as well